advisories/advisory-side.php

```diff
@@ -1,15 +1,11 @@
-<br/><p>ownCloud server 9.0.3</p>
-<a href="/security/advisory?id=oc-sa-2016-010">Insecure Direct Object References in Gallery</a><br/>
```

Comment:
010 is still there - but on the side bar we list only advisoriees which are relevant for the latest versions